prime-router/docs/getting-started/fhir-functions.md

```diff
@@ -1,7 +1,68 @@
+both extend an interface called FhirPathFunctions. There are two places for these functions because some
+are internal functions, meaning that they can only be called from within the code rather than from the command line.
 
-- Custom FHIR Patch Functions
-  - Internal
-    - Just LIVD Lookup
-  - External
-    - See the CustomFHIRFunctions.kt file for full list
+## Internal FHIR Path Functions
+There is currently only one internal FHIR path function and that is the LIVD Lookup function which can be found [here](../../src/main/kotlin/fhirengine/engine/CustomFhirPathFunctions.kt)
+
+- livdTableLookup - This function must be called on a resource of type `Observation` with a parameter specifying 
+    the field in the `LivdLookupTable` that you want to search on. If the `deviceId` on the 
+    Observation is set, it will do a lookup on the `deviceId` for the parameter specified. If the `deviceId` is not present
+    and the `equipmentModelId` is present, it will search on the `equipmentModelId`. If both of those are not present, 
+    but the `deviceName` is, it will search for the `equipmentModelName`. What is returned are any results that match 
+    the criteria found in the LIVD table. A preview of this table can be found in the `LIVD-SARS-CoV-2.csv` file.
+
+## External FHIR Path Functions
+There are many external FHIR path functions which can be found [here](../../src/main/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctions.kt)
+
+An example of one of the currently most complicated functions is the convertDateToAge function:
+- convertDateToAge(<optional> timeUnit, <optional> comparisonDate) - The resource must be a date. Since this function 
+required a lot of code to complete, it is broken out into a separate helper function. This is the pattern we hope to 
+use moving forward. This required so much extra code partially because it takes multiple, optional params which can be
+in any order. One can specify whether this value is returned in years, months, or days. 
+If left off, the method assumes years if it is greater than one year, months if it is less than a year but 
+greater than one month, and days if the value is less than one month. There is also an optional param to pass a 
+comparison date if you don't want to get the age based off of how old they are today.
+(timeUnit, comparisonDate) and (comparisonDate, timeUnit) are both acceptable. It returns an age in years, months, or 
+days.Example Output: 
+>{  
+>"value": "DecimalType[31]"  
+>"unit": "year" &nbsp;  
+>"code": "a"  
+>}  
+>Number of results = 1 ----------------------------
+
+## How to define a FHIR path function
+1. Determine which file it will go in by reading the [internal](#internal) and [external](#external) sections of this 
+document.
+2. Add the function itself.
+3. Add it to the resolveFunction method. This registers it so the command line knows what to expect and is valid.
```

Comment:
```suggestion
3. Add it to the `resolveFunction` method. This registers it so the command line knows what to expect and is valid.
```

Let's be more consistent with what we put in code blocks (I have a hard time with this too) and try to have every method name in these quotes.